portfolio/src/main/webapp/style.css

```diff
@@ -41,3 +40,18 @@ button {
   padding-left: 150px;
   padding-top: 50px;
 }
+
+#experienceSection {
+    margin-top: 30px;
```

Comment:
Nit: Another way to write this by using

`
margin: 30px 0
`
^ this will allow margin top and margin bottom as 30px, but margin left and right as 0